styles/style.css

```diff
@@ -1,9 +1,19 @@
 /*Header styles*/
+@font-face {
+  font-family: "Roboto";
+  src: url("../about/fonts/Roboto-Medium.woff") format("woff");
+}
+@font-face {
+  font-family: "Roboto";
+  src: url("../about/fonts/Roboto-Regular.woff") format("woff");
+}
 .header {
   width: 1440px;
   height: 569px;
 }
 .text {
+  font-family: Roboto-Medium;
+  font-style: normal;
   padding: 120px 567px 0 567px;
```

Comment:
you can left only padding-top:120px
and add property text-align:center
It is will be look much better)) Also try to use quantities aliqout of 5 i.e If you measured on layout size 123 you can write it like 125